projects/plugins/protect/src/js/data/account-protection/use-toggle-account-protection-module-mutation.ts

```diff
@@ -15,14 +16,23 @@ export default function useToggleAccountProtectionMutation(): UseMutationResult
 
 	return useMutation( {
 		mutationFn: API.toggleAccountProtection,
-		onMutate: () => {
+		onMutate: async () => {
```

Comment:
```suggestion
		onMutate: () => {
```
What does this change do?

Reply:
Unintended! Not sure how that got in there 🤔